### DISCUSSION TITLE
Analysis of Transcriptome data by SQK-PCB109

### DISCUSSION THREAD

**Discussion by ramnageena11** [felixgrunberger/microbepore#3] [General]
Hi,
I am establishing a Omics lab with ONT and transcriptome is one the minimum explored/studied area on ONT for microbes. I saw your methodology and workflow and excited to use it for my analysis. I am bit new and not a expert in programming but trying to do all my analysis.

I have generated a transcriptome data from a bacterium in two conditions using ONT. I have sequenced genome (ONT), assembled (flye) and annotated by prokka. Prokka generates several files after annotation such as .fasta for genome, .fasta for genes (nucleotides and aminoacids), .gff. .gbk, .sqn etc. 

##Can I use .gff file generated by prokka to make transcript.fasta? or
##.fasta for genes will work for = transcript.fasta?
## could I be able to perform differential analysis?

I was going through the pipeline/ workflow and noticed that quality control was performed after mapping. why so?
why not pychopper was used after the read qc and demux stage?

I would appreciate for suggestions and support.
Thanks
RNS

**Comment by ramnageena11** [comment]
Hi,
I would appreciate if you look into my querries.

Thanks
RNS

**Comment by felixgrunberger** [comment]
Hi, 
Regarding your first question: You can use the gene .fasta as transcripts file. However, you might look up the prokka doc to check whether it only contains coding or also non-coding transcripts.
For DGE, ONT also offers pipelines that may be worth looking at (https://github.com/nanoporetech/pipeline-transcriptome-de), especially if you are starting things off. 
In the repository workflow, QC is performed on raw reads and after mapping, to improve identification of read boundaries, which is not necessary for DGE. 
In any way, I would recommend to run pychopper to be able to perform DGE in a strand-specific manner. 

Cheers,
Felix

**Reply by ramnageena11** [reply]
Hi,
Thank Felix.
im following Microbepore pipeline first. I will see the results and then go for transcriptome-de.

rgds
Ram
